### PR TITLE
Style navbar

### LINKED ISSUE #289: Style navigation

### PR DESCRIPTION
**Pull request by dejvid15**
Closes #289

Removed app version (should be included in the footer in the new design), will change the colors after merging the PR with colors definitions, removed old navigation styles, button should be styled after merging the buttons PR.

### CODE REVIEW

File: app/frontend/components/common/navbar/navbarUnauth.js [hunk 1]
```diff
@@ -45,14 +47,14 @@ const NavbarUnauth = connect(
         h(
           'button',
           {
-            class: 'demo-button rounded-button',
+            class: 'button outline navbar-button',
```
Comment (darjanin) [inline app/frontend/components/common/navbar/navbarUnauth.js]: Please, remove `navbar-button`. I thought you wrote that we will have 2 types of buttons. Normal and small. I will stick to it at the moment. If you want to have another version of the `.button`, it should be defined next to the `.button` declarations, not next to the `.nav`.

File: app/frontend/components/common/navbar/navbarUnauth.js [hunk 2]
```diff
@@ -36,7 +38,7 @@ const NavbarUnauth = connect(
         h(
           'a',
           {
-            class: 'unauth-menu',
+            class: 'navbar-link',
             href: 'https://github.com/vacuumlabs/adalite/wiki/AdaLite-FAQ',
             target: '_blank',
```
Comment (darjanin) [inline app/frontend/components/common/navbar/navbarUnauth.js]: Add `rel: 'noopener'` to any link that has`target: '_blank'`.

File: app/frontend/components/common/navbar/navbarUnauth.js [hunk 3]
```diff
@@ -10,24 +9,27 @@ const NavbarUnauth = connect(
   actions
 )(({openGenerateMnemonicDialog}) =>
   h(
```
Comment (darjanin) [inline app/frontend/components/common/navbar/navbarUnauth.js]: I was thinking more like this usage of `h1`.
```js
 h(
    'nav',
    {class: 'navbar'},
    h(
      'div',
      {class: 'navbar-wrapper'},
      h(
        'h1',
        {class: 'navbar-heading'},
        h('span', {class: 'navbar-title'}, 'AdaLite - Cardano Wallet'),
        h(
          'a',
          {class: 'navbar-logo', href: '/'},
          h('img', {
            src: 'assets/adalite-logo.svg',
            alt: 'AdaLite - Cardano Wallet',
          })
        )
      ),

```

and the styles for `.navbar-title`

```css
.navbar-title {
  width: 0;
  height: 0;
  text-indent: -999999px;
  display: inline-block;
}
```

File: app/public/css/styles.css [hunk 4]
```diff
@@ -1577,3 +1308,36 @@ td {
...
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.navbar-heading {
+  display: none;
+}
+
+.navbar-link {
```
Comment (darjanin) [inline app/public/css/styles.css]: Please add `:hover` version of `.navbar-link`.

File: app/public/css/styles.css [hunk 5]
```diff
@@ -1577,3 +1308,36 @@ td {
...
+.navbar-wrapper {
+  max-width: var(--width);
+  width: 100%;
+  margin: 0 auto;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.navbar-heading {
```
Comment (darjanin) [inline app/public/css/styles.css]: This can be removed at the moment.

File: app/public/css/styles.css [hunk 6]
```diff
@@ -1577,3 +1308,36 @@ td {
...
+.navbar-link {
+  margin-right: 40px;
+  font-size: 14px;
+  line-height: 32px;
+  text-decoration: none;
+  /* TODO: replace with var --grey after colors PR merge */
+  color: #606a71;
+}
+
+.navbar-button {
```
Comment (darjanin) [inline app/public/css/styles.css]: I would remove it at the moment.

File: app/frontend/components/common/navbar/navbarUnauth.js (hunk 1 above)
Comment (dejvid15) [reply app/frontend/components/common/navbar/navbarUnauth.js]: We discussed that we won't have 3 versions for the "common" button as intended by the design but we'll simplify that for now by having just 2 common button versions. The "special" button that we've discussed is the navbar button which remains intact in terms of styling so  I'll move the `navbar-button` styling to the buttons portion of the file.

File: app/frontend/components/common/navbar/navbarUnauth.js (hunk 2 above)
Comment (dejvid15) [reply app/frontend/components/common/navbar/navbarUnauth.js]: will do

File: app/public/css/styles.css (hunk 4 above)
Comment (dejvid15) [reply app/public/css/styles.css]: Designer decision was that no hover effect should be applied to the links, specifically asked about this as it seemed strange to me at first, that's why there's no `:hover` declaration

File: app/public/css/styles.css (hunk 6 above)
Comment (dejvid15) [reply app/public/css/styles.css]: Removed

File: app/public/css/styles.css (hunk 5 above)
Comment (dejvid15) [reply app/public/css/styles.css]: Removed

File: app/frontend/components/common/navbar/navbarUnauth.js (hunk 3 above)
Comment (dejvid15) [reply app/frontend/components/common/navbar/navbarUnauth.js]: Great :)

**Comment by dejvid15** [comment]
@darjanin Implemented your review comments, I'd still wait for PR #287 before merging

File: app/public/css/styles.css (hunk 4 above)
Comment (darjanin) [reply app/public/css/styles.css]: What's the reason for not having no `:hover` on links?

**Comment by dejvid15** [comment]
@darjanin Replaced color vars from PR #287, rebased & added `:hover` state for `navbar-link`

**Pull request merged**